docs/resources/antiddos_basic.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Anti-DDoS"
+---
+
+# huaweicloud_antiddos_basic
```

Comment:
basic is an adj, suggest to use a noun here.

Reply:
basic can also be a noun, we can get the product name from [service overview](https://support.huaweicloud.com/intl/en-us/productdesc-ads/ads_01_0001.html)
```
HUAWEI CLOUD provides multiple security solutions to defend against Distributed Denial-of-service (DDoS) attacks. 
You can select an appropriate one based on your service requirements. HUAWEI CLOUD Anti-DDoS Service (ADS) 
includes Cloud Native Anti-DDoS Basic (CNAD Basic), Cloud Native Anti-DDoS Advanced (CNAD Advanced), and
Advanced Anti-DDoS (AAD).
```

Reply:
basic can also be a noun, we can get the product name from [service overview](https://support.huaweicloud.com/intl/en-us/productdesc-ads/ads_01_0001.html)
```
HUAWEI CLOUD provides multiple security solutions to defend against Distributed Denial-of-service (DDoS) attacks. 
You can select an appropriate one based on your service requirements. HUAWEI CLOUD Anti-DDoS Service (ADS) 
includes Cloud Native Anti-DDoS Basic (CNAD Basic), Cloud Native Anti-DDoS Advanced (CNAD Advanced), and
Advanced Anti-DDoS (AAD).
```